dockerfiles/che/entrypoint.sh

```diff
@@ -199,6 +199,24 @@ launch_docker_registry () {
     fi
 }
 
+perform_database_migration() {
+  CHE_DATA=/data
+  # Move database to original location
+
+  if [ -f ${CHE_DATA}/storage/db/che.mv.db ]; then
+    echo "Performing migration of the database file, to it's original old path"
+    echo "In case if there was already a database file in old path, it will be renamed to 'che.mv.db.old'"
+    echo "See issue https://github.com/eclipse/che/issues/8068 for details"
+    #in case if there is existing database in old path, back it up
+    if [ -f ${CHE_DATA}/db/che.mv.db ]; then
+      mv ${CHE_DATA}/db/che.mv.db ${CHE_DATA}/db/che.mv.db.old
```

Comment:
It can be useful to add logs that `old` database is backed up

Reply:
ok